AUTHORS

```diff
@@ -2064,5 +2064,82 @@ Waleed Ashraf <waleedashraf@outlook.com>
 Mir Mufaqam Ali <mannanali413@gmail.com>
 Nicholas Drane <nicholasdrane@gmail.com>
 Shobhit Chittora <chittorashobhit@gmail.com>
+Dmitriy Kasyanov <korvin1986@gmail.com>
+Tyson Andre <tandre@themeetgroup.com>
+Salame William <william.salame@student.uclouvain.be>
+Todd Wong <toddwong@leftart.com>
+Mykola Bilochub <nbelochub@gmail.com>
+Qingyan Li <qingyan.liqy@alibaba-inc.com>
+Sho Miyamoto <shqld8@gmail.com>
+现充 <qixiang.cqx@alibaba-inc.com>
+furstenheim <furstenheim@gmail.com>
+Moritz Peters <maritz@users.noreply.github.com>
+laino <lain@volafile.io>
+Suixinlei <joshuasui@163.com>
+wandalen <wandalen.me@gmail.com>
+Mars Wong <marswong618@gmail.com>
+Omar Crisostomo <omarcb@micorreo.upp.edu.mx>
+leeseean <leeseean@qq.com>
+Gilles De Mey <gilles.de.mey@gmail.com>
+Matthew Turner <matty_t47@hotmail.com>
+ryanmahan <ryanmahan97@gmail.com>
+Antony Tran <antonyt@gmail.com>
+Robert Adamian <robert.adamyan1@gmail.com>
+Peter Dalgaard-Jensen <nephross.cortex@gmail.com>
+Seth Brenith <sethb@microsoft.com>
+Adam Engebretson <adam@enge.me>
+Aaron Kau <aaron.kau@rutgers.edu>
+Ivan Filenko <ivan.filenko@protonmail.com>
+Hackzzila <admin@hackzzila.com>
+Timothy O. Peters <timotewpeters@gmail.com>
+Ben Wilcox <reccanti@gmail.com>
+Justin Ridgewell <justin@ridgewell.name>
+jvelezpo <sebasvelez@gmail.com>
+Guangcong Luo <guangcongluo@gmail.com>
+Sarat Addepalli <sarat.addepalli@paytm.com>
+iskore <iSkore@users.noreply.github.com>
+Camilo Gonzalez <cgs.camilogonzalez@gmail.com>
+BufoViridis <30531954+BufoViridis@users.noreply.github.com>
+Bhavani Shankar <ebhavanishankar@gmail.com>
+JiaHerr Tee <jiaherrt@gmail.com>
+Jeff Principe <princjef@gmail.com>
+alejandro estrada <estrada9166@gmail.com>
+Aonghus O Nia <aonghusonia@eircom.net>
+Mihail Bodrov <connormiha1@gmail.com>
+Chin Huang <chhuang@us.ibm.com>
+Kevin Caulfield <kevincaulfield01@gmail.com>
+Sergey Golovin <golovim@gmail.com>
+Miles Elam <miles.elam@productops.com>
+juggernaut451 <singhjug1994@gmail.com>
+killagu <killa123@126.com>
+dustinnewman98 <dustinnewman98@gmail.com>
+Alejandro Estrada <estrada9166@gmail.com>
+flickz <omoyajowo2015@gmail.com>
+Wilson <wla80@sfu.ca>
+Eric Bickle <wolf.code@outlook.com>
+Ujjwal Sharma <usharma1998@gmail.com>
+Wei-Wei Wu <wuxx1045@umn.edu>
+punteek <prateek1@ucla.edu>
+Tobias Nießen <tniessen@users.noreply.github.com>
+Ken Lin <ken23421@gmail.com>
+Piotr Grzesik <pj.grzesik@gmail.com>
+feugy <damien.feugas@gmail.com>
```

Comment:
The name is "Damien Feugas" and should be reflected as such (he has a middle name but I guess that is not important?).